Dockerfile

```diff
@@ -12,10 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Initialized TARGETPLATFORM with default value with assumption that the docker build happens on the ppc64le arch
+ARG TARGETPLATFORM=linux/ppc64le
```

Comment:
I see docker build PR job is failing, we need to change this to `limux/amd64` or we need to pass an appropriate TARGETPLATFORM in the make image.

I prefer making this `linux/amd64` as our CI environment is amd64 based arch.

Reply:
fixed